docs/src/architectures/alchemical-model.rst

```diff
@@ -59,6 +59,15 @@ hyperparameters to tune are (in decreasing order of importance):
   This hyperparameter controls the size and depth of the descriptors and the neural
   network. In general, increasing this might lead to better accuracy,
   especially on larger datasets, at the cost of increased training and evaluation time.
+- ``loss_weights``: This controls the weighting of different contributions to the loss
```

Comment:
Maybe mention that missing entries in `loss_weights` will be weighted by 1.0.

Reply:
This is mentioned above in the hypers documentation

Reply:
While this section is the "what to change" part